packages/@aws-cdk/aws-cognito/lib/user-pool-attr.ts

```diff
@@ -341,3 +341,250 @@ export class DateTimeAttribute implements ICustomAttribute {
     };
   }
 }
+
+/**
+ * This interface contains all standard attributes recognized by Cognito
+ * from https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html
+ * including `preferred_email` and `preferred_phone_number`
```

Comment:
Curious: why are preferred_email and preferred_number_number special? 
They don't feature as standard attributes in the cognito docs. why are they considered as standard?

Reply:
My bad. It is not `preferred_email` and `preferred_phone_number` to be special, it is `email_verified` and `phone_number_verified`.

These are _special_ because are not directly mentioned in the list at [the linked page](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html) but are present in the cited [OpenID Connect Specification](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims) and are in fact present among the other standard attributes.

Reply:
> are in fact present among the other standard attributes

What do you mean by this? Were you able to confirm that they are actually supported by Cognito, but just missing in the documentation?

If that's the case, we should just add these to `StandardAttributes`.

Reply:
I have conducted some tests and I confirm [Configuring User Pool](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html) is missing two attributes: `email_verified` and `phone_number_verified`.

Even though it states:

> You get a set of default attributes, called "standard attributes," with all user pools. You can also add custom attributes to your user pool definition in the AWS Management Console. This topic describes those attributes in detail and gives you tips on how to set up your user pool.

and does not list them under standard attributes, the following call:

```bash
❯ aws cognito-idp create-user-pool --cli-input-json file://userpool.json
❯ cat userpool.json 
{
    "PoolName": "test",
    "Policies": {
        "PasswordPolicy": {
            "MinimumLength": 10,
            "RequireUppercase": true,
            "RequireLowercase": true,
            "RequireNumbers": true,
            "RequireSymbols": true,
            "TemporaryPasswordValidityDays": 10
        }
    },
    "AutoVerifiedAttributes": [
        "email"
    ],
    "UsernameAttributes": [
        "email"
    ],
    "Schema": [
        {
            "Name": "email",
            "AttributeDataType": "String",
            "Mutable": true,
            "Required": true,
            "StringAttributeConstraints": {
                "MinLength": "5",
                "MaxLength": "2048"
            }
        },
        {
            "Name": "email_verified",
            "AttributeDataType": "Boolean",
            "Mutable": true,
            "Required": true
        }
    ],
    "UsernameConfiguration": {
        "CaseSensitive": true
    }
}
```
works as expected, setting both `email` and `email_verified` as _required_ and _mutable_ attributes.

The UI Console has the same problem as it does not show `(phone_number|email)_verified` in the attribute list:
<img width="811" alt="image" src="https://user-images.githubusercontent.com/8103786/104584957-a54d7500-5663-11eb-8bc0-a30cacb83d68.png">

_I filed feedbacks for the documentation and the web ui pages._

That said, I will add those 2 attributes to the `StandardAttributes` interface, so `StandardAttributes` and `StandardAttributesMask` will have the same properties.